ppl/src/main/java/org/opensearch/sql/ppl/PPLService.java

```diff
@@ -95,4 +108,20 @@ private PhysicalPlan plan(PPLQueryRequest request) {
         .plan(logicalPlan);
   }
 
+  private void setConnector(UnresolvedPlan unresolvedPlan) {
```

Comment:
I think connect should be in CatalogService, PPLService should not ware it.

Reply:
Didnt get your suggestion here? Are you suggesting to pass unresolved plan to catalog service and get the relevant storage and execution engines?